### PR TITLE
use snakemake and IntervalMetadata

### PR DESCRIPTION
**Pull request by RNAer**
it depends on the branch of scikit-bio: https://github.com/rnaer/scikit-bio/tree/micronota, where the genbank parser and gff3 parser is waiting to be merged to skbio.

### CODE REVIEW

File: micronota/rules/config.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        threads: 1
+    # aragorn:
+    #     params: '-l'
+    #     priority: 50
+    #     threads: 1
+    # diamond:
+    #     params: '--index-chunks 1 --query-cover 80 -k 3'
+    #     priority: 50
+    #     threads: 1
+    #     db: '/Users/zech/Dropbox/git/micronota/micronota/data'
```
Comment (mortonjt) [inline micronota/rules/config.yaml]: wut?

File: micronota/rules/config.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    prodigal:
+        # params: '-p meta'
+        params: '-p single'
+        priority: 90
+        threads: 1
+    # cmscan:
+    #     params: ''
+    #     priority: 50
+    #     threads: 1
+    #     db: '/Users/zech/Dropbox/git/micronota/micronota/data/RF00169.cm'
```
Comment (mortonjt) [inline micronota/rules/config.yaml]: wut?

**Comment by mortonjt** [comment]
damn - there are no tests here ...

We can't have this sort of test coverage on the master branch.  Let's create a developer branch - so we can more easily sort out the outstanding tasks ...

**Comment by RNAer** [comment]
it is just because of the snakemake files. The coverage will increase when more code comes in.

**Comment by mortonjt** [comment]
right - which is why we should be pushing this into a development branch, not the master branch.

We did this strategy with Emperor.  This won't slow down development.

**Pull request closed without merging**